src/interfaces/models.ts

```diff
@@ -11,12 +11,6 @@ import {
 
 export type SubmissionTypeT = 'examination' | 'consent' | 'normal'
 
-export interface AnalysisJSONI {
-  header?: string
-  issues: IssueI[]
-  status: 'fa' | 'rc' | 'ar'
-}
-
```

Comment:
I moved this below the declaration for `IssueI`.

Ditto for other changes in here.